tt1-blocks/assets/css/style-shared.css

```diff
@@ -23,49 +23,3 @@ a:hover {
 	text-decoration-style: dotted;
 	text-decoration-skip-ink: none;
 }
-
-/*
- * Gutenberg remotes this underline, but Twenty Twenty-One uses it. 
- */
-.site-header h1.wp-block-site-title a:not(:hover):not(:focus):not(:active) {
-	text-decoration: underline;
-}
-
-/*
- * Alignment styles.
```

Comment:
We will add the theme.json for the global padding on [this issue](https://github.com/WordPress/community-themes/issues/25#issuecomment-1490110408)